README.md

```diff
@@ -15,29 +15,39 @@ Before running DICOMFuse, ensure that you've installed the following
 applications:
 
 *   Java SE Runtime Environment 8.
-*   [WinFps](https://github.com/billziss-gh/winfsp) for using FUSE in Windows.
+*   [WinFps](https://github.com/billziss-gh/winfsp) for using FUSE in Windows 
+    (Windows 7x64, Windows 10x64).
 *   [OSXFuse](https://osxfuse.github.io/) for using FUSE in macOS (macOS 10.14
     Mojave).
+*   [libfuse](https://github.com/libfuse/libfuse) for using FUSE in Linux, install 
+    libfuse - `sudo apt install libfuse2`
 
-> [libfuse](https://github.com/libfuse/libfuse) has been supported in Linux
-> kernel since 2.6.14 version.
+> FUSE has been supported in Linux kernel since 2.6.14 version.
 
 ### Credentials
 
 You can access the Cloud Healthcare API using
 [Google application default credentials](https://cloud.google.com/docs/authentication/production#howtheywork).
+Set Healthcare DICOM Editor role for 
```

Comment:
Switch this paragraph to:

DICOMFuse will use [application default credentials](https://cloud.google.com/docs/authentication/production#howtheywork) to authenticate access to the Cloud Healthcare API. The identity must have the permissions within the [Healthcare DICOM Editor role](https://cloud.google.com/healthcare/docs/concepts/access-control#roles). 

Alternatively you can specify a service account key via the `--keyFile` flag.

The easiest way to setup credentials is to use your user credentials with the [gcloud tool](http://cloud/sdk/gcloud/):

```
gcloud auth application-default login
```

Reply:
Done